tests/integration/test_definitions/sqlite/integration_tests.py

```diff
@@ -0,0 +1,55 @@
+from tests.integration.backend_dependencies import BackendDependencies
+from tests.integration.integration_test_fixture import IntegrationTestFixture
+
+sqlite_integration_tests = []
+
+connecting_to_your_data = [
```

Comment:
Glad we are tracking the work involved to make this `--sqlite` rather than `--sqlalchemy`.